src/xenia/base/system_gnulinux.cc

```diff
@@ -21,6 +23,8 @@
 // Use headers in third party to not depend on system sdl headers for building
 #include "third_party/SDL2/include/SDL.h"
 
+extern char** environ;
```

Comment:
Is `extern "C"` not required for it? By the way, since it's needed only once, I think it's better to declare it at the local scope where it's used, so there won't be a conflict if some other header also declares `environ`.